apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -244,6 +246,8 @@ class OppgaveService(
 
         if (frist != null) {
             oppgaveDao.redigerFrist(oppgave.id, frist)
+        } else {
+            oppgaveDao.redigerFrist(oppgave.id, toDagerFremITid)
```

Comment:
Så vidt jeg ser kan denne _alltid_ settes til to dager frem i tid. Trenger ikke IF-en i det hele tatt siden det kun er ett sted frist blir satt, og der settes den også til  to dager frem i tid.

Reply:
Så det samme, men lurte på om metoden tok inn frist av en grunn og at 'noen hadde tenkt'. Men du mener det er bedre å fjerne frist som input?

Reply:
Endret som foreslått